From 8310f89d8818bc9d41b386bbb2824593aa8e8dca Mon Sep 17 00:00:00 2001
From: Auger Eric <eric.auger@redhat.com>
Date: Wed, 12 Aug 2020 11:03:31 +0200
Subject: RHEL-only: Enable vTPM for ARM in downstream configs

RH-Author: Auger Eric <eric.auger@redhat.com>
Message-id: <20200811163601.14341-3-eric.auger@redhat.com>
Patchwork-id: 98144
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH 2/2] RHEL-only: Enable vTPM for ARM in downstream configs
Bugzilla: 1801242

We allow the compilation of the TPM_TIS_SYSBUS device and both
passthrough and software emulation backends.

Signed-off-by: Eric Auger <eric.auger@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 default-configs/aarch64-rh-devices.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/default-configs/aarch64-rh-devices.mak b/default-configs/aarch64-rh-devices.mak
index f0cf5a1b22..0b6a03f816 100644
--- a/default-configs/aarch64-rh-devices.mak
+++ b/default-configs/aarch64-rh-devices.mak
@@ -20,3 +20,6 @@ CONFIG_VIRTIO_PCI=y
 CONFIG_XIO3130=y
 CONFIG_NVDIMM=y
 CONFIG_ACPI_APEI=y
+CONFIG_TPM_EMULATOR=y
+CONFIG_TPM_PASSTHROUGH=y
+CONFIG_TPM_TIS_SYSBUS=y
-- 
2.27.0

